migrations/20170818-origin-rename.sql

```diff
@@ -0,0 +1,9 @@
+/* Add copyright header here.
```

Comment:
Oh right -- yeah, I took the header from another SQL file.  Who owns the copyright on this?